ichub-backend/controllers/fastapi/app.py

```diff
@@ -78,3 +88,29 @@ async def partner_management_create_business_partner(business_partner_create: Bu
 @app.get("/partner-management/business-partner/{business_partner_number}/data-exchange-agreement", response_model=List[DataExchangeAgreementRead], tags=["Partner Management"])
 async def partner_management_get_data_exchange_agreements(business_partner_number: str) -> List[DataExchangeAgreementRead]:
     return partner_management_service.get_data_exchange_agreements(business_partner_number)
+
+@app.get("/twin-management/catalog-part-twin", response_model=List[CatalogPartTwinRead], tags=["Twin Management"])
+async def twin_management_get_catalog_part_twins(include_data_exchange_agreements: bool = False) -> List[CatalogPartTwinRead]:
+    return twin_management_service.get_catalog_part_twins(include_data_exchange_agreements=include_data_exchange_agreements)
+
+@app.get("/twin-management/catalog-part-twin/{global_id}", response_model=List[CatalogPartTwinDetailsRead], tags=["Twin Management"])
+async def twin_management_get_catalog_part_twin(global_id: UUID) -> List[CatalogPartTwinDetailsRead]:
+    return twin_management_service.get_catalog_part_twin_details(global_id)
+
+@app.post("/twin-management/catalog-part-twin", response_model=TwinRead, tags=["Twin Management"])
+async def twin_management_create_catalog_part_twin(catalog_part_twin_create: CatalogPartTwinCreate) -> TwinRead:
+    return twin_management_service.create_catalog_part_twin(catalog_part_twin_create)
+
+@app.post("/twin-management/catalog-part-twin/share", responses={
+    201: {"description": "Catalog part twin shared successfully"},
+    204: {"description": "Catalog part twin already shared"}
+}, tags=["Twin Management"])
```

Comment:
```suggestion
}, tags=["Share Management"])
```

Could we group the share-related APIs into a separate group? It would make them easier to identify in the OpenAPI documentation.

Reply:
I already talked with @sm29105, and this will be changed in a future PR.